src/Compiler/Checking/MethodOverrides.fs

```diff
@@ -427,8 +427,9 @@ module DispatchSlotChecking =
                         else
                             // Error will be reported below in CheckOverridesAreAllUsedOnce 
                             ()
-                    | _ -> 
-                        fail(Error(FSComp.SR.typrelOverrideWasAmbiguous(FormatMethInfoSig g amap m denv dispatchSlot), m))
+                    | ambiguousOverrides ->
```

Comment:
Instead of using `List.head` below, I think this should rather be `| ambigousOverride :: _ ->`. I mean, as far as I can tell, we know for sure the list is not empty anyway, but it is much cleaner IMO.